.github/workflows/ci.yml

```diff
@@ -106,66 +143,10 @@ jobs:
         build_dir: docs/_build/html
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
   pass:
     needs: [test]
     runs-on: ubuntu-latest
     steps:
       - run: echo "All jobs passed"
-  deploy-images:
```

Comment:
"4. Removing Containerization: Removed containerization-related files and configurations from the repository. This simplifies the project setup and reflects the decision to move away from container-based workflows."

Who's decision and when was this discussed? I'm of the opinion that this is a very nice feature to have.

Reply:
Does the container based workflow work currently?